requirements.txt

```diff
@@ -42,7 +42,7 @@ retrying
 rq-scheduler
 shapely
 sendgrid==6.1.0
-sentry-sdk==0.11.2
+sentry-sdk==0.12.3
 shippo==2.0.0rc1
```

Comment:
Why did you change the version here?

Reply:
I was getting an error that was crashing my worker. It traced back to an issue with the Sentry token. The bug was fixed with an upgrade.